bin/config-yargs.js

```diff
@@ -14,53 +14,46 @@ module.exports = function(yargs) {
 		.alias("help", "h")
 		.version()
 		.alias("version", "v")
+		// These commands are not actually handled by yargs, as webpack.js
+		// pre-emptively looks for them prior to yargs processing. This is
+		// strictly for --help output
+		.command(
+			"init",
+			"Initializes a new webpack configuration or loads a addon if " +
+			"specified",
+		)
+		.command(
+			"migrate",
+			"Migrate your webpack configuration from webpack 1 to webpack 2"
+		)
+		.command(
+			"add",
+			"Adds a webpack component to your configuration file"
+		)
```

Comment:
You can suppress `add` command as well.

https://github.com/webpack/webpack-cli/blob/4f8ebee5f2397162488ec82c179593c118cf1c2d/bin/webpack.js#L24

Reply:
Suppress meaning?

Reply:
Comment. :grin:

Reply:
I don't see the problem here

Reply:
In `master`, the add command [is not commented out](https://github.com/webpack/webpack-cli/blob/4f8ebee5f2397162488ec82c179593c118cf1c2d/bin/config-yargs.js#L32-L36). I thought it best not to change existing behavior with this PR